sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java

```diff
@@ -2014,39 +1878,36 @@ private Response<BinaryData> beginAzureBatchImageGenerationWithResponse(
      *     }
      * }
      * }</pre>
-     *
+     * 
      * @param imageGenerationOptions Represents the request data used to generate images.
      * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @return the {@link PollerFlux} for polling of a polling status update or final response payload for an image
-     *     operation.
+     * operation.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<BinaryData, BinaryData> beginBeginAzureBatchImageGenerationAsync(
-            BinaryData imageGenerationOptions, RequestOptions requestOptions) {
-        return PollerFlux.create(
-                Duration.ofSeconds(1),
-                () -> this.beginAzureBatchImageGenerationWithResponseAsync(imageGenerationOptions, requestOptions),
-                new DefaultPollingStrategy<>(
-                        new PollingStrategyOptions(this.getHttpPipeline())
-                                .setEndpoint("{endpoint}/openai".replace("{endpoint}", this.getEndpoint()))
-                                .setContext(
-                                        requestOptions != null && requestOptions.getContext() != null
-                                                ? requestOptions.getContext()
-                                                : Context.NONE)
-                                .setServiceVersion(this.getServiceVersion().getVersion())),
-                TypeReference.createInstance(BinaryData.class),
-                TypeReference.createInstance(BinaryData.class));
+    public PollerFlux<BatchImageGenerationOperationResponse, BatchImageGenerationOperationResponse>
```

Comment:
The return of the `PollerFlux` change from `<BinaryData, BinaryData>` to `<BatchImageGenerationOperationResponse, BatchImageGenerationOperationResponse>` . We don't generally have types that aren't `BinaryData` as return types for the HTTP client implementation. Is this an expected change on the emitter?

Reply:
This seems be diff tool artifact.

The `beginBeginAzureBatchImageGenerationAsync` is still here.
https://github.com/Azure/azure-sdk-for-java/blob/57a197bede8c970623247f939a10361141383125/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java#L1763-L1774

This one is `beginBeginAzureBatchImageGenerationWithModelAsync` (note "WithModel"). The reason is that there is very hard to convert `SyncPoller<BinaryData, BinaryData>` to model flavor `SyncPoller<BatchImageGenerationOperationResponse, BatchImageGenerationOperationResponse>`. Hence codegen had to generate another LRO "WithModel"